src/_pytest/fixtures.py

```diff
@@ -1529,34 +1530,49 @@ def sort_by_scope(arg_name: str) -> int:
         return initialnames, fixturenames_closure, arg2fixturedefs
 
     def pytest_generate_tests(self, metafunc: "Metafunc") -> None:
+        """Generate new tests based on parametrized fixtures used by the given metafunc"""
+
+        def get_parametrize_mark_argnames(mark: Mark) -> Sequence[str]:
```

Comment:
We might as well inline it, unless the line becomes too ling.

Reply:
I would, but it is clunky because `ParameterSet._parse_parametrize_args` returns a tuple, and we are only interested in the first one:

```python
if any(
    argname in ParameterSet._parse_parametrize_args(*mark.args, **mark.kwargs)[0]
    for mark in metafunc.definition.iter_markers("parametrize")
):
```